src/pages/schedule-interview/components/DesktopCandidateSelectInterviewTime/components/DesktopInterviewTimeSlotBooking/DesktopInterviewTimeSlotBooking.tsx

```diff
@@ -0,0 +1,106 @@
+
+  const headerRender = ({ value, onChange }: { value: Dayjs; onChange: (date: Dayjs) => void }) => (
+    <div className="mb-4 flex items-center justify-between px-5">
+      <LeftOutlined
+        onClick={() => onChange(value.clone().subtract(1, "month"))}
+        className="text-secondary cursor-pointer text-xl"
+      />
+      <span className="px-2 text-base font-semibold text-black">{`${value.format("MMMM")} ${value.year()}`}</span>
+      <RightOutlined
+        onClick={() => onChange(value.clone().add(1, "month"))}
+        className="text-secondary cursor-pointer text-xl"
+      />
+    </div>
+  )
+
+  const fullCellRender = (value: Dayjs) => (
+    <div
+      className={cn([
+        "ant-picker-cell-inner ant-picker-calendar-date rounded-full text-center",
+        isInterviewDate(value) ? "text-secondary bg-[rgba(0,0,0,0.04)] text-base" : "text-sm text-black/50",
+        currentDate === formatDayjs(value) && "bg-sapia-pink",
+      ])}
+      onClick={() => handleDateChange(value)}
+    >
+      <div className="rounded-full">{value.date()}</div>
+    </div>
+  )
+
+  return (
+    <div data-testid="desktop-interview-time-slot-booking" className={cn([styles.interviewTimeSlotBookingWrapper])}>
+      <section className="flex justify-between">
+        <section>
+          <Calendar
+            className="interviewTimeSlotBookingCalendar w-[316px]"
+            fullscreen={false}
+            headerRender={headerRender}
+            fullCellRender={fullCellRender}
+            disabledDate={disabledDate}
+          />
+          <p className="mb-0 mt-3 text-left text-xs font-light text-black">Timezone: {userTimezone}</p>
+        </section>
+        <section className="ml-10 w-[350px]">
+          <div className="flex items-center justify-between">
+            <div className="text-lg font-semibold">{formatDateToLongString(interviewDate.toDate())}</div>
+            <div className="text-sm font-normal text-black/65">
+              {timeSlots.length} time slot{timeSlots.length > 1 ? "s" : ""}
+            </div>
+          </div>
+          <div className="h-[350px] overflow-y-auto">
+            <Radio.Group
+              className="w-full"
+              onChange={(e) => handleTimeSlotChange(e.target.value as string)}
+              value={selectedTime}
+            >
+              {timeSlots.map((slot: string) => (
+                <Radio.Button key={slot} value={slot} className="w-full px-5 text-left text-black">
+                  {selectedDateAndTime.selectedTime === slot && selectedDateAndTime.selectedDate === currentDate && (
+                    <CheckCircleFilled className="checkCircleFilled mr-3 rounded-full bg-[#FFCEFF] text-black" />
+                  )}
+                  {slot}
+                </Radio.Button>
+              ))}
+            </Radio.Group>
+          </div>
+        </section>
+      </section>
+      <Button
+        className={cn([
+          "buttonBook mt-4 w-full text-base font-normal",
```

Comment:
Same question here, why we need `buttonBook` here?
```suggestion
          "mt-4 w-full text-base font-normal",
```

I saw you use this in the css module to locate the button and then change the styles, but I don't think that's how it should work
![image](https://github.com/PredictiveHire/phapp-schedule-link/assets/7485288/bedd5137-ef2c-4db2-a141-a1c6ebfa063d)

If you do need another class to locate the button or change the precedence, you can try to take it out of `:global()`, and then use it as `styles.buttonBook` in the component
![image](https://github.com/PredictiveHire/phapp-schedule-link/assets/7485288/611135d1-b10b-45f8-a9d4-81e1391ea693)
![image](https://github.com/PredictiveHire/phapp-schedule-link/assets/7485288/49e31559-0050-4c7a-835f-63b1c7a3c60b)

By doing so, we can make sure each of these selectors are well-scoped